Paws4Me/Paws4Me/Controllers/Logic/AllPetDetailViewController.swift

```diff
@@ -35,27 +35,27 @@ class AllPetDetailViewController: UIViewController, UISearchBarDelegate {
     @IBAction private func catTappedButton(_ sender: UIButton) {
            getPetTypeFromButton(sender)
            sender.changePetIconsColor()
-           searchPetType(type: petType)
+           searchPetByCategoryType(petCategoryType: animalType)
```

Comment:
I would suggest declaring this function like this:

`func searchPetCategoryType(_ petCategoryType)`

Then when you call it it should look like this: 

`searchPetCategoryType()`

please note I also removed the "by" normally we try to avoid and small words that are not really important for the function name 😄

Reply:
Done 👍🏻